src/helpers/links.js

```diff
@@ -38,8 +38,12 @@ const domHref = function(node, relativePath) {
 		return ref
 	}
 	// Don't rewrite links in collectives app context
-	if (loadState('core', 'active-app') === 'collectives') {
-		return ref
+	try {
+		if (loadState('core', 'active-app') === 'collectives') {
+			return ref
+		}
+	} catch {
+		// throws error in public shares
```

Comment:
loadState also supports a default which looks a bit cleaner

```suggestion
	if (loadState('core', 'active-app', '') === 'collectives') {
		return ref
```

Reply:
Makes much more sense :see_no_evil:

Thanks, changed now.